api/DOMTokenList.json

```diff
@@ -30,7 +30,7 @@
             "version_added": "11.5"
           },
           "safari": {
-            "version_added": "5.1"
+            "version_added": "6"
           },
           "safari_ios": {
             "version_added": "5"
```

Comment:
Does this also need to be updated to 6? https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#browser_compatibility has a range starting at 5, but that seems implausible, that might just be wrong too.

Reply:
It seems to be just mirroring the original desktop data.  Fixing now!